addons/account/wizard/account_move_reversal.py

```diff
@@ -103,10 +103,13 @@ def reverse_moves(self):
             action.update({
                 'view_mode': 'form',
                 'res_id': moves_to_redirect.id,
+                'context':{'default_type':  moves_to_redirect.type},
             })
         else:
             action.update({
                 'view_mode': 'tree,form',
                 'domain': [('id', 'in', moves_to_redirect.ids)],
             })
+            if len(set(moves_to_redirect.mapped('type'))) == 1:
+                action['context'] = {'default_type':  moves_to_redirect.mapped('type')}
```

Comment:
Sorry but in case of you have 2 moves you will have something like that:
`['in_refund', 'in_refund']` and it will raise an error if you click on the create button. 
Please fix that.

Reply:
@FlorianGilbert very clever. I have added a pop function in order to make it a string. I believe it would do it